Packs/BmcHelixRemedyForce/IncidentTypes/incidenttype-Bmc_Remedyforce.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In the demo session we talked about 2 incident type (named "Incident" and "Service Request") that should be renamed to "BMC Remedyforce Incident" and "BMC Remedyforce Service Request".
I see only one incident type here.

Reply:
As discussed on call, we have one single underlying Salesforce entity for IN/SR, so we have kept a one common layout along with one common incident type.
Yes for label changes for integration configuration param for incident type would be taken care of.

Reply:
As discussed on call again, we would prepare two separate layoputs/incident types for IN and SR both.

Reply:
We have added two different incident types for incident and service request.